pycbc/events/stat.py

```diff
@@ -1476,7 +1479,12 @@ def single(self, trigs):
             # exists for SingleDetTriggers
             self.curr_tnum = trigs['template_id']
             # Should only be one ifo fit file provided
-            assert len(self.ifos) == 1
+
+            # Arthur Comment: I can understand the need for the assertion
```

Comment:
@GarethCabournDavies This comment is a question I have right now. The assertion exists in the `single` method so I can understand why you would assert that you need only 1 ifo. But this code is only called when a coincident detection is made (I believe). Therefore if it is _not_ commented out I get this error: 

`  File "/home/arthur.tolley/.conda/envs/live_rank_stat/lib/python3.9/site-packages/pycbc/events/stat.py", line 1486, in single
    assert len(self.ifos) == 1
AssertionError`

and when it is commented out the code runs through. Should I remove this code? I don't know what the harm would be in removing it.

Reply:
This seems to be a little bit of legacy - if calling through `rank_method.single()` when using SingleDetTriggers, this is used, but I don't see anywhere in the code (after a quick browse) which uses this, as most places seem to call direct to get_sngl_ranking.

I think it should be okay to update the comment a line above and remove this assertion.

Reply:
Okay cool, I have removed both the comment and assertion (unless you wanted me to keep the comment for something else?)

Reply:
my 'update the comment' referred to the `exists for SingleDetTriggers` comment above. If you could add to that to say the type that you're using now as well, that would be good